### PR TITLE
tidy5: update checksum

### PR DESCRIPTION
**Pull request by ajtcode71**
#### Testing the changes
updated checksum per issue #39072. added grep depend to get to build.

- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (X86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl
  - aarch64
  - armv7l
  - armv6l
  - armv6l-musl
  - i686
  - x86_64-musl

### CODE REVIEW

File: srcpkgs/tidy5/template [hunk 1]
```diff
@@ -6,13 +6,13 @@ wrksrc="tidy-html5-${version}"
 build_style=cmake
 configure_args="-DTIDY_CONSOLE_SHARED=ON"
 cmake_builddir=build/cmake
-hostmakedepends="libxslt"
+hostmakedepends="libxslt grep"
```
Comment (classabbyamp) [inline srcpkgs/tidy5/template:9]: why is grep needed?

Comment (ajtcode71) [reply srcpkgs/tidy5/template:9]: *] Updating repository `https://ftp.swin.edu.au/voidlinux/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating software in / masterdir...
/void-packages/common/xbps-src/shutils/chroot.sh: line 51: grep: command not found
=> xbps-src: cleaning up / masterdir...

build breaks further down as it doesnt have grep.   I have bootstrapped masterdir. Im now wondering if the mirror is missing files?

Comment (paper42) [reply srcpkgs/tidy5/template:9]: https://github.com/void-linux/void-packages/pull/39575#discussion_r985083246

Comment (ajtcode71) [reply srcpkgs/tidy5/template]: fixed my masterdir.
removed grep.

**Pull request merged**